lib/mail_room/connection.rb

```diff
@@ -16,8 +16,6 @@ def wait
       raise NotImplementedError
     end
 
-    def quit
-      raise NotImplementedError
-    end
+    def quit; end
```

Comment:
@tpitale I made it optional because only the IMAP connection appears to need anything special to `quit` because it uses a separate thread to idle. The Microsoft Graph connection doesn't need to do anything.

We could either make this the default implementation or make the Microsoft Graph implementation implement `quit`.

Reply:
This is fine. I just needed to understand.